src/workingWithData.ts

```diff
@@ -0,0 +1,54 @@
+import { v4 as uuidv4 } from 'uuid'
+import { resolve } from 'path'
+import { readFile, open, writeFile, unlink, access, mkdir } from 'fs/promises'
+
+type UserInfoType = { name: string; age: number }
+
+export const createFolderIfNotExist = async function (path: string) {
+  const pathToDIr: string = resolve(path)
+  try {
+    await access(pathToDIr)
+  } catch (error) {
+    await mkdir(pathToDIr)
+    console.log('The dir has been created')
+  }
+}
+
+export const createContent = async function (
```

Comment:
Please specify return type

Reply:
also you in this case more better don't use arrow function
`export async function createContent(...`

Reply:
also you in this case more better don't use arrow function ----- Also would be better if you don't use arrow function in this case
Done